### ISSUE TITLE
HA 2022.7.0 - Addon 'xxx' does not exist but they still backup

### ISSUE THREAD

**Issue by mjnovotny82** [jcwillox/hass-auto-backup#62]
At least from what I can recall, this error started after upgrading to HA 2022.7.0 but it is backing up the addons.... weird.

> This error originated from a custom integration.
> 
> Logger: custom_components.auto_backup
> Source: custom_components/auto_backup/__init__.py:264
> Integration: Auto Backup (documentation, issues)
> First occurred: 12:41:04 PM (6 occurrences)
> Last logged: 12:41:04 PM
> 
> Addon 'core_mosquitto' does not exist
> Addon 'a0d7b954_zwavejs2mqtt' does not exist
> Addon 'f1c878cb_eufy-ha-mqtt-bridge' does not exist
> Addon 'a0d7b954_vscode' does not exist
> Addon 'a0d7b954_nodered' does not exist

> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'core_mariadb' does not exist
> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'core_mosquitto' does not exist
> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'a0d7b954_zwavejs2mqtt' does not exist
> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'f1c878cb_eufy-ha-mqtt-bridge' does not exist
> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'a0d7b954_vscode' does not exist
> 2022-07-07 12:41:04 WARNING (MainThread) [custom_components.auto_backup] Addon 'a0d7b954_nodered' does not exist

![Backup Entry](https://user-images.githubusercontent.com/90118616/177836804-3ad23aac-fc8c-4561-8d42-a85772e46544.PNG)

```
automation:
  - id: perform_auto_backup
    alias: Perform Auto Backup
    trigger:
      platform: time_pattern # Perform backup every 3 hours.
      hours: "/4"
    condition:
      - condition: time
        after: "05:30:00"
        before: "21:30:00"
    action:
      service: auto_backup.backup_partial
      data:
        name: "AutoBackup: {{ now().strftime('%a, %-I:%M %p (%Y%m%d)') }}" # %d/%m/%Y
        addons:
          - core_mariadb # It doesn't matter if you use the addon slug or name. Name is easier.
          - core_mosquitto
          - a0d7b954_zwavejs2mqtt
          - f1c878cb_eufy-ha-mqtt-bridge
          - a0d7b954_vscode
          - a0d7b954_nodered
        folders:
          - homeassistant
          - share
          - ssl
        keep_days: 2
```

**Comment by jcwillox** [comment]
Hmm, yes I seem to have introduced a bug in the last version, I'll try to get it fixed soon. I wouldn't worry about it as it should just be an incorrect warning and everything should still work normally.

**Comment by jcwillox** [comment]
This issue has been fixed in release [1.1.1](https://github.com/jcwillox/hass-auto-backup/releases/tag/1.1.1).
